Source/ProjectRimFactory/Common/HarmonyPatches/Patch_DSU_OrbitalTrade.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RimWorld;
+using HarmonyLib;
+using Verse;
+using ProjectRimFactory.Storage;
+using System.Reflection.Emit;
+using System.Reflection;
+
+namespace ProjectRimFactory.Common.HarmonyPatches
+{
+    //This patch allows Dsu's to Act as a trade beachon
```

Comment:
I would break this into two files:
Patch_TradeUtility_AllLaunchablethingsForTrade_forDSU.cs
Patch_PassingShip_toMakeDSUTradeBeacon.cs

When debugging others' code, it's super helpful to have the patched class in the filename.  I didn't do that with Deep Storage, and I think it was the wrong choice.